e2e-tests/local-testnet-singlechain.sh

```diff
@@ -202,7 +202,6 @@ done
 PROVIDER_NODE_LISTEN_ADDR_STR=${PROVIDER_NODE_LISTEN_ADDR_STR::${#PROVIDER_NODE_LISTEN_ADDR_STR}-1}
 PROV_NODES_HOME_STR=${PROV_NODES_HOME_STR::${#PROV_NODES_HOME_STR}-1}
 
-echo "Testnet applications are set up! Run the following command to start CometMock:"
```

Comment:
nit: could still keep a message that CometMock has started

Reply:
Thanks, good point